extensions/library-genesis/README.md

```diff
@@ -1,4 +1,5 @@
 <p align="center">
+  <img src="assets/command-icon.png" height="128">
```

Comment:
@yz3440 this unfortunately won't work atm, we have a bug that doesn't let images using their relative path in html code render. You need to use the absolute path from github or markdown lang to render the image

Reply:
Thanks for letting me know!